### ISSUE TITLE
use miniimagenet

### ISSUE THREAD

**Issue by qiangqiangqiang123** [DebabrataPal7/MORGAN#1]
Since your paper title contains few-shot open-set recognition,can we use miniimagenet and tieredimaganet to test your model?

**Comment by DebabrataPal7** [comment]
> Since your paper title contains few-shot open-set recognition,can we use miniimagenet and tieredimaganet to test your model?

Certainly, you can use it. Just change the feature extractor to tackle 2d images instead of the Hyperspectral feature extractor. In fact, we used MORGAN on the MiniImageNet dataset with the ResNet-18 feature extractor for another OSR project benchmarking purpose, https://github.com/DebabrataPal7/DAFOSNET

**Comment by qiangqiangqiang123** [comment]
> > Since your paper title contains few-shot open-set recognition,can we use miniimagenet and tieredimaganet to test your model?
> 
> Certainly, you can use it. Just change the feature extractor to tackle 2d images instead of the Hyperspectral feature extractor. In fact, we used MORGAN on the MiniImageNet dataset with the ResNet-18 feature extractor for another OSR project benchmarking purpose, https://github.com/DebabrataPal7/DAFOSNET

Actually,we have followed your articles(MORGAN & DAFOSNET),but DAFOSNET is domain adaptive setting and we hope to conduct your generative based method on open-set setting. It's a little bit difficult for me to modify your code since it is tensorflow rather than pytorch and your dataset(HSI) process is quite different from other method.Would you mind giving me several advice?
